core/src/main/java/org/ehcache/StandaloneCacheBuilder.java

```diff
@@ -60,17 +61,19 @@ public StandaloneCacheBuilder(final Class<K> keyType, final Class<V> valueType,
   }
 
   T build(ServiceLocator serviceLocator) {
-    Store.Provider storeProvider = serviceLocator.findService(Store.Provider.class);
-    if (serializationProvider == null) {
-      this.serializationProvider = serviceLocator.findService(SerializationProvider.class);
+    try {
+      serviceLocator.startAllServices(new HashMap<Service, ServiceConfiguration<?>>());
+    } catch (Exception e) {
+      logger.error("StandaloneCacheBuilder failed due to {}.", e.getMessage());;
```

Comment:
The exception should be logged as such, not its message (ie: logger.error("some message", exception) instead of  logger.error("some message: ", exception.getMessage) ) to make sure the logger can print the full stack trace.